test/integration/objectstore/_base.py

```diff
@@ -26,6 +27,18 @@
 """
 )
 
+ONEDATA_DEMO_SPACE_NAME = "demo-space"
+ONEDATA_OBJECT_STORE_CONFIG = string.Template("""
+<object_store type="onedata">
+    <auth access_token="${access_token}" />
+    <connection onezone_domain="${onezone_domain}" verify_ssl="False"/>
+    <space name="${space_name}" />
```

Comment:
jeśli wprowadzisz opcję path to warto by ją tu wytestować, wtedy bym proponował
```
<space name="${space_name}" ${optional_space_params} />
```
a następnie wylosować w setupie testu że `optional_space_params` ma jedną z trzech wartości:
```
''
'path=""'
'path="a/b/c/d"'
```
trzeba by jeszcze opisać w conf.sample czy path ma się zaczynać od slasha czy nie i jaki jest default